packages/css/package.json

```diff
@@ -42,7 +42,7 @@
     "SICK Design System"
```

Comment:
Do we also need to adapt the naming here and say "Synergy Design System" instead of "SICK Design System"? Or maybe just add it as keyword? Same question for the other package.json files